scripts/validate_oas_examples.py

```diff
@@ -51,6 +54,19 @@
 # List all endpoints
 all_endpoints_url = oas_spec_from_yaml["paths"].keys()
 
+for endpoint in all_endpoints_url:
+    all_http_methods = oas_spec_from_yaml["paths"][endpoint]
+    for http_method in all_http_methods:
+        endpoint_spec = oas_spec_from_yaml["paths"][endpoint][http_method]["$ref"]
+        try:
+            with open(os.path.join(spec_path, endpoint_spec), "r") as spec_yaml_file:
+                oas_spec_map[endpoint_spec] = safe_load(spec_yaml_file)
+        except FileNotFoundError:
+            msg = "File or path does not exist. Consider executing " "'make publish'."
+            print(msg, endpoint_spec)
+            raise
```

Comment:
Assuming this is to build a dictionary with all yamls that have been refactored from the main one, I think a FileNotFoundError would mean a spec for an endpoint is missing, so make publish would fail. So maybe this error message could be changed to something like
`"File or path does not exist. An OAS is missing for endpoint " + endpoint` ?